package.json

```diff
@@ -65,9 +63,9 @@
     "ci:test": "npm run test",
     "clean": "rm -rf typechain-types cache bin dist artifacts contracts-exposed",
     "postclean": "hardhat clean",
+    "collector:deploy": "hardhat deploy-collector",
     "compile": "npm run clean && hardhat compile --force",
     "deploy": "hardhat deploy --network",
```

Comment:
Agree with not cluttering the package json with scripts. Shouldn't the 'deploy' script also be removed from here?

Reply:
I wasn't unsure about it. WDYT?

Reply:
It falls under the same category as all other tasks, so yes, we could remove it.